packages/ui/registry/src/UiWalletRegistry_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.ts

```diff
@@ -0,0 +1,112 @@
+import type { IdentifierArray, Wallet } from '@wallet-standard/base';
+import type { UiWalletHandle, UiWalletAccount } from '@wallet-standard/ui-core';
+
+import { getOrCreateUiWalletAccountForStandardWalletAccount_DO_NOT_USE_OR_YOU_WILL_BE_FIRED } from './UiWalletAccountRegistry_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.js';
+import { registerWalletHandle_DO_NOT_USE_OR_YOU_WILL_BE_FIRED } from './UiWalletHandleRegistry_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.js';
+import { identifierArraysAreDifferent } from './compare.js';
+
+type UiWallet = UiWalletHandle &
+    Readonly<
+        Pick<Wallet, 'chains' | 'icon' | 'name' | 'version'> & {
+            accounts: readonly UiWalletAccount[];
+        }
+    >;
```

Comment:
Why does this not import the `UiWallet` type?

Reply:
Refactor fumble. ty!